splinterd/src/config/mod.rs

```diff
@@ -194,4 +257,24 @@ mod tests {
         // Compare the generated PartialConfig object against the expected values.
         assert_config_values(built_config);
     }
+
+    #[cfg(feature = "config-default")]
```

Comment:
This should go in the config::default::test module.

Reply:
Addressed.